packages/devtools/lib/src/utils.dart

```diff
@@ -147,6 +147,30 @@ String longestFittingSubstring(
 bool isLetter(int codeUnit) =>
     (codeUnit >= 65 && codeUnit <= 90) || (codeUnit >= 97 && codeUnit <= 122);
 
+/// Returns a simplified version of a StackFrame name.
+///
+/// Given an input such as
+/// "_WidgetsFlutterBinding&BindingBase&GestureBinding.handleBeginFrame", this
+/// method will strip off all the Mixin names and return
+/// "_WidgetsFlutterBinding.handleBeginFrame".
```

Comment:
Nit: use ` in dartdoc comments instead of " as comments are interpreted as markdown.

Reply:
Also file a bug on the dart vm to stop showing these  `&` names from the VM and reference that bug here.

Reply:
Issue already filed for this problem: https://github.com/dart-lang/sdk/issues/36999.